Various technical notes/transition_infos.json

```diff
@@ -576,15 +571,9 @@
     },
     {
       "area_id": "0xDE524DA6",
-      "area_name": "Twin Outpost (Underwater Cave)",
+      "area_name": "Twin Outposts (Underwater Cave)",
       "default_entrance": "0xE6B9138A",
-      "exits": [
-        {
-          "area_id": "0xE6B9138A",
-          "area_name": "Twin Outposts (2 exits)",
-          "requires": []
-        }
-      ]
+      "exits": []
     },
```

Comment:
Same comment as below. Either keep the exits, or remove it from transitions entirely with a comment.

Reply:
As mentioned in both the commit titles as well as the utils.py file that writes it to the Spoiler Log:
**Twin Outposts (Underwater Cave)** is temporarily removed from the randomization proccess. An essential part of that removal is that this exit is not included in the .json file.

Reply:
I mark this as resolved because it's a duplicate of another conversation.